tests/test_views.py

```diff
@@ -402,7 +402,7 @@ def test_head_and_get_headers_match_with_identifier(self, rf):
 class TestAppEvent:
     """Tests for views.app_event."""
     CONTENT_TYPE = 'application/atom+xml'
-    RESULTS_PER_PAGE = 20
+    RESULTS_PER_PAGE = views.EVENT_SEARCH_PER_PAGE
```

Comment:
Here you are using `EVENT_SEARCH_PER_PAGE`, in `app_event` you hard code 200. Do we want to switch in one place or the other?